hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -131,6 +134,9 @@ public class VerifyReplication extends Configured implements Tool {
    */
   public static class Verifier extends TableMapper<ImmutableBytesWritable, Put> {
 
+    private ThreadPoolExecutor reCompareExecutor = null;
```

Comment:
This Executor and its helper methods are only used from within the Map task, so moved them in here. It's possible that some MR environment will instantiate multiple map task instances per JVM, which would result in all tasks sharing the same pool, but I don't think that's how MRv2 works.